src/amphtml-email/20_Samples/Advanced_Server_Request.html

```diff
@@ -66,29 +43,31 @@
     {{/animal}}
   </template>
   <!--
-    The `amp-list` is displayed and loads the initial data from our server endpoint, while the
-    `amp-form` is initially hidden.
+    The `amp-list` loads the initial data from our server endpoint.
+    When the user makes a selection, the `amp-list` becomes hidden and the `amp-form` displays
```

Comment:
nit: avoid passive voice. Better: 
```
When the user changes the selection, the change event triggers the form submission and hides the `amp-list`: `on="change:animal-form.submit,animal-list.hide"`. The `amp-form` renders the server response using the same template as ...
```

The other thing is: it's better to combine actions that belong together.